docs/standard/assembly/names.md

```diff
@@ -17,19 +18,19 @@ myTypes, Version=1.0.1234.0, Culture=en-US, PublicKeyToken=b77a5c561934e089c, Pr
 ```
 
 > [!NOTE]
-> Processor architecture is added to the assembly identity in the .NET Framework version 2.0, to allow processor-specific versions of assemblies. You can create versions of an assembly whose identity differs only by processor architecture, for example 32-bit and 64-bit processor-specific versions. Processor architecture is not required for strong names. For more information, see <xref:System.Reflection.AssemblyName.ProcessorArchitecture%2A?displayProperty=nameWithType>.
+> Processor architecture is added to the assembly identity in .NET Framework 2.0, to allow processor-specific versions of assemblies. You can create versions of an assembly whose identity differs only by processor architecture, for example 32-bit and 64-bit processor-specific versions. Processor architecture is not required for strong names. For more information, see <xref:System.Reflection.AssemblyName.ProcessorArchitecture%2A?displayProperty=nameWithType>.
```

Comment:
I feel as though we should cull .NET Framework 2.0 from notes where they are calling out the difference between 1.0 and 2.0. .NET Framework evolved quite a lot from 1->2 but it's all standard now. Considering 2.0 support ended in 2011, I think we can safely omit this.

If .NET Framework 2.0 is removed, I think this note loses it's purpose and can be removed.